src/tv2-common/actions/CoreActionExecutionContext.ts

```diff
@@ -0,0 +1,238 @@
+import {
+	BlueprintMappings,
+	IActionExecutionContext,
+	IBlueprintMutatablePart,
+	IBlueprintPart,
+	IBlueprintPartInstance,
+	IBlueprintPiece,
+	IBlueprintPieceDB,
+	IBlueprintPieceInstance,
+	IBlueprintResolvedPieceInstance,
+	PackageInfo,
+	Time
+} from 'blueprints-integration'
+import { literal, PartMetaData, PieceMetaData } from 'tv2-common'
+import { ITV2ActionExecutionContext } from './context'
+
+export class CoreActionExecutionContext implements ITV2ActionExecutionContext {
+	public studioId: string
+	public isTV2Context: true = true
```

Comment:
This existed to prevent passing the original core context around instead of the decorated one, but the way things are structured now, I don't think that can ever happen. removed